### DISCUSSION TITLE
Missing plugin in Alpha 2

### DISCUSSION THREAD

**Discussion by wezetel** [trobfab/fabrik#415] [Q&A] [answered]
In Alpha 2 some plugins, which are mentioned as working, are not included. Is there a way to add them manually? And how is this done from github.
In fact I want to test an application I have build with F3 and now want to re-create it with F4. The plugin I'm missing is 'time'.

**Comment by wezetel** [comment]
As I have just seen from the plugin status list the time plugin will be included in Alpha 3. So I just wait ...

**Comment by skurvish** [comment, marked as answer]
Fixed in alpha3
